packages/cli/src/Generate.ts

```diff
@@ -50,6 +50,7 @@ ${bold('Options')}
       --schema   Custom path to your Prisma schema
        --watch   Watch the Prisma schema and rerun after a change
    --generator   Generator to use (may be provided multiple times)
+   --no-engine   Generate a client for use with Accelerate only
```

Comment:
For now this is its only purpose, but we might use it for other purposes in the future.